src/docs/truffle/getting-started/interacting-with-your-contracts.md

```diff
@@ -202,7 +202,7 @@ When you make a transaction, you're given a `result` object that gives you a wea
 * `result.logs` *(array)* - Decoded events (logs)
 * `result.receipt` *(object)* - Transaction receipt
 
-For more information, please see the [README](https://github.com/trufflesuite/truffle-contract) in the `truffle-contract` project.
+For more information, please see the [README](https://github.com/trufflesuite/truffle/tree/develop/packages/truffle-contract) in the `truffle-contract` project.
```

Comment:
For this can we link directly to the readme?

[Here](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-contract/README.md)